modules/products/components/product-actions.tsx

```diff
@@ -0,0 +1,55 @@
+"use client";
+import { useState } from "react";
+
+export default function ProductPriceAndActions({
+  price,
+  stock,
+  productId,
+}: {
+  price: number;
+  stock: number;
+  productId: string;
+}) {
+  const [quantity, setQuantity] = useState(1);
+
+  const quantityOptions = [];
+  for (let i = 1; i <= stock; i++) {
+    quantityOptions.push(i);
+  }
+
+  const handleAddToCart = () => {
+    const newItem = { productId, quantity };
+    alert(`Producto agregado: ${JSON.stringify(newItem)}`);
```

Comment:
```suggestion
//TODO: cambiar alert por toast
    alert(`Producto agregado: ${JSON.stringify(newItem)}`);
```
un toast de hero ui para luego